### PR TITLE
MQE-683: Assert action deprecation + upgrade

### PR DESCRIPTION
**Pull request by KevinBKozan**
### TODO:
* Create CE/EE/B2B/Ext branches and merge into 3.0.0 branch

### CODE REVIEW

File: src/Magento/FunctionalTestingFramework/Upgrade/UpdateAssertionSchema.php [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        $testsPath = $input->getArgument('path');
+        $finder = new Finder();
+        $finder->files()->in($testsPath)->name("*.xml");
+
+        $fileSystem = new Filesystem();
+        $testsUpdated = 0;
+        foreach ($finder->files() as $file) {
+            $contents = $file->getContents();
+            // Isolate <assert ... /> but never <assert> ... </assert>, stops after finding first />
+//            preg_match_all('/<assert[^>]*\/>/', $contents, $potentialAssertions);
```
Comment (soumyau) [inline src/Magento/FunctionalTestingFramework/Upgrade/UpdateAssertionSchema.php]: commented code can be removed?

Comment (KevinBKozan) [reply src/Magento/FunctionalTestingFramework/Upgrade/UpdateAssertionSchema.php]: Will do 👍

File: src/Magento/FunctionalTestingFramework/Util/TestGenerator.php [hunk 2]
```diff
@@ -623,9 +623,6 @@ public function generateStepsPhp($actionObjects, $generationScope = TestGenerato
             } elseif (isset($customActionAttributes['url'])) {
                 $input = $this->addUniquenessFunctionCall($customActionAttributes['url']);
                 $url = $this->addUniquenessFunctionCall($customActionAttributes['url']);
-            } elseif (isset($customActionAttributes['expectedValue'])) {
```
[the remark this answers is not in the document]
Comment (KevinBKozan) [reply src/Magento/FunctionalTestingFramework/Util/TestGenerator.php]: 1. I don't think so; we flatten nested format for standardization of attributes in `ActionObject`. I'l double check this claim.
2. Yes, that can be removed for sure.

[the remark this answers is not in the document]
Comment (KevinBKozan) [reply src/Magento/FunctionalTestingFramework/Util/TestGenerator.php]: Confirmed 1. can't be removed 👍

**Pull request merged**